pex/interpreter_constraints.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# A library of functions for filtering Python interpreters based on compatibility constraints
+
+from .common import die
+from .interpreter import PythonIdentity
+from .tracer import TRACER
+
+
+def validate_constraints(constraints):
+  # TODO: add check to see if constraints are mutually exclusive (bad) so no time is wasted
```

Comment:
Did this TODO exist already when you moved it? If not, can we start an issue for it to track?

Reply:
✅